Add issue template for communication

mbbroberg opened the pull request:
- Creates a way to request support from Upstream Marketing Group

Signed-off-by: Matthew Broberg <1744971+mbbroberg@users.noreply.github.com>

<!--  Thanks for sending a pull request!  Here are some tips for you:
- If this is your first contribution, read our Getting Started guide https://github.com/kubernetes/community/blob/master/contributors/guide/README.md
- If you are editing SIG information, please follow these instructions: https://git.k8s.io/community/generator
  You will need to follow these steps:
  1. Edit sigs.yaml with your change 
  2. Generate docs with `make generate`. To build docs for one sig, run `make WHAT=sig-apps generate`
-->

/sig contributor-experience
/area contributor-comms

mbbroberg commented:
/cc @parispittman 
/assign @jonasrosland 

Hey gang, I took a first swing at the PR template we've been talking about and based it off of the others out there. Big questions: 

- Flow: I'd love to ask as few questions as possible in the right order. I really like how some of the ones Paris suggested original get people to think about their goals and separately ask for copy.
- Assignement: I'm not sure if it's helpful or harmful to have you autoassigned Jonas. Given your as Internal Comms it makes sense, but I want to be respectful of your workflow. 

Thanks!

mrbobbytables commented on .github/ISSUE_TEMPLATE/marketing-request.md:
```diff
@@ -0,0 +1,53 @@
+---
+name: Marketing Request
+about: Get help getting a message out to our community through coordinated outreach no email, social media, and other public channels.
```
```suggestion
about: Get help getting a message out to our community through coordinated outreach on email, social media, and other public channels.
```
no -> on 👍

mrbobbytables commented on .github/ISSUE_TEMPLATE/marketing-request.md:
```diff
@@ -0,0 +1,53 @@
+about: Get help getting a message out to our community through coordinated outreach no email, social media, and other public channels.
+title: 'REQUEST: New communication about <topic>'
+labels: area/contributor-comms, sig/contributor-experience
+assignees: '@jonasrosland'
```
hm -- it may not be best to assign to one person for bus-factor. Maybe ping a github team instead? would need to create one in k/org, but everyone would be notified of the request and folks could self-assign.

mbbroberg replied:
Sounds good - did we land on a team?

mrbobbytables commented on .github/ISSUE_TEMPLATE/marketing-request.md:
```diff
@@ -0,0 +1,53 @@
+assignees: '@jonasrosland'
+---
+<!--
+ATTENTION: the Upstream Marketing Team have classifications on how urgent and important a message to the community is [outlined here][multichannel].
```
```suggestion
ATTENTION: the Upstream Marketing Team have classifications on how urgent and important a message to the community is outlined here: https://git.k8s.io/community/communication/marketing-team/multichannel-communications.md#mapping-type-to-tier
```
It might be best to just link to the doc directly here, as its a comment it won't appear either when creating the issue off of it or in the issue after creation.

mrbobbytables commented on .github/ISSUE_TEMPLATE/marketing-request.md:
```diff
@@ -0,0 +1,53 @@
+
+
+**What is the sponsoring SIG or community group?**
+<!-- -->
```
```suggestion
<!-- What SIG or group is this message on behalf of? -->
```
Maybe something like that in there?

mrbobbytables commented on .github/ISSUE_TEMPLATE/marketing-request.md:
```diff
@@ -0,0 +1,53 @@
+- [ ] No, but I'd prefer <$DATE> <$TIME>
+
+**Where do you want it to be visible?**
+<!-- Note that email via our mailinst list is the default choice for all messages. See our [multichannel communication strategy][multichannel] for more details. Note that blogs and social media must meet our guidelines available [here](https://github.com/kubernetes/community/tree/master/communication/marketing-team#purpose). -->
```
```suggestion
<!-- Note that email via our mailing list is the default choice for all messages. See our [multichannel communication strategy][multichannel] for more details. Note that blogs and social media must meet our guidelines available [here](https://github.com/kubernetes/community/tree/master/communication/marketing-team#purpose). -->
```
Like the earlier link this one won't appear as a markdown link and needs to be text =/

Maybe something like:
```
Note that email via our mailing list is the default choice for all messages. See our
multichannel communication strategy for more details. Note that blogs and social
media must meet our guidelines.

Multichannel communication strategy: http://git.k8s.io/community/communication/marketing-team/multichannel-communications.md#mapping-type-to-tier

Blog and Social Media Guidelines: https://git.k8s.io/community/communication/marketing-team#purpose
```

mbbroberg replied:
Great point!

parispittman commented:
once links are fixed and the other small nits, we are ready to merge

mbbroberg commented on .github/ISSUE_TEMPLATE/marketing-request.md:
```diff
@@ -0,0 +1,53 @@
+---
+name: Marketing Request
```
```suggestion
name: Contributor Comms Request
```

parispittman commented on .github/ISSUE_TEMPLATE/marketing-request.md:
```diff
@@ -0,0 +1,53 @@
+- [ ] Blog post (with help from a [Storyteller][team])
+- [ ] Twitter ([@k8scontributors](https://twitter.com/k8scontributors)
+- [ ] YouTube ([Kubernetes](https://www.youtube.com/channel/UCZ2bu0qutTOM0tHYa_jkIwg))
+- [ ] Slack (assumes `#announcements` unless otherwise mentioned)
```
```suggestion
- [ ] Slack (assumes `#kubernetes-dev` unless otherwise mentioned)
```

mbbroberg commented:
Thanks @parispittman @mrbobbytables -- all updates included and fixed up. I've removed the assignment for now. We can decide if we need a group assignment of if the labels are sufficient for us to track. Take another look when you can.

mbbroberg commented:
@mrbobbytables mentioned that teams cannot be assigned, and it's not ideal to make a new team or single assignment. This current proposal is what we'd like to move forward on.

mrbobbytables commented:
/approve
/hold
/assign @nikhita 
just to get 1 last set of eyes on the template format etc 👍 
Everything looks good to me though :)

nikhita commented:
Lgtm :)

mrbobbytables commented:
/hold cancel
/pony party

An lgtm from someone on the marketing team and this should be good to go 👍

mbbroberg commented on .github/ISSUE_TEMPLATE/marketing-request.md:
```diff
@@ -0,0 +1,58 @@
+---
+name: Contributor Comms Request
+about: Get help getting a message out to our community through coordinated outreach on email, social media, and other public channels.
```
```suggestion
about: Get support on sending a message out to our community through coordinated outreach on email, Slack, social media, and other public channels.
```

mbbroberg replied:
Removing redundancy.

mbbroberg commented on .github/ISSUE_TEMPLATE/marketing-request.md:
```diff
@@ -0,0 +1,58 @@
+-->
+
+**What do you want send out?** 
+<!-- Please include a link whenever possible, logos, artwork, pics, diagrams, or any other assets with public links. -->
```
```suggestion
<!-- Include logos, artwork, pics, diagrams, or any other assets with public links whenever possible. -->
```

mbbroberg commented on .github/ISSUE_TEMPLATE/marketing-request.md:
```diff
@@ -0,0 +1,58 @@
+<!-- What SIG or group is this message on behalf of? What is the best way to contact them? -->
+
+**Is this time-sensitive?** 
+<!-- You can delete or skip this if it's not relevant -->
```
```suggestion
<!-- You can delete or skip this if it's not relevant. -->
```

Consistent punctuation.

mbbroberg commented on .github/ISSUE_TEMPLATE/marketing-request.md:
```diff
@@ -0,0 +1,58 @@
+- [ ] Other (fill this out)
+
+**Where is your first draft available?**
+<!-- Please provide a first draft of your message for all channels requested. Take your best shot and we are here to support you through getting it ready for publication (it's appreciated). 
```
```suggestion
<!-- Please provide a first draft of your message for all channels requested. Take your best shot and we are here to support you through getting it ready for publication (it's appreciated). Links to Google Docs or Hackmd.io pages are best for easy editing.
```

mbbroberg commented on .github/ISSUE_TEMPLATE/marketing-request.md:
```diff
@@ -0,0 +1,58 @@
+**Lastly, who's the best point of contact?** 
+<!-- If you are the point of contact, please confirm it's you. Please include both GitHub username and Slack username. -->
+
+
```
```suggestion
<!-- Last but not least, thank you! We're excited to help get the word out. -->
```

parispittman commented:
/lgtm

Woohoo
